pom.xml

```diff
@@ -173,6 +173,12 @@
         <!-- Version suffix that is appended to directories. Default is the maven GAV version but this can be edited to use a short form version -->
         <server.output.dir.version>${project.version}</server.output.dir.version>
 
+        <!-- Protocol to use for communication with remote maven repositories.
+             You can set to 'http' if you are using a maven proxy and 'https' 
+             interferes with that. Use 'https' for builds that will be released
+             to non-snapshot public maven repos -->
+        <maven.repository.protocol>https</maven.repository.protocol>
```

Comment:
I personally think that ability to control whole url is better (gives you chance to use different repo), OTOH for the purpose of being able to use http protocol to be able to use maven proxy this is perfectly valid

Reply:
I'll add a second property, which uses this one for the protocol part of its default value.

Reply:
@psakar Done